onnxscript/type_annotation.py

```diff
@@ -25,60 +27,31 @@
 _list_constructors = [list, typing.List, typing.Sequence]
 
 
-def _get_origin(t: type) -> typing.Optional[type]:
-    """Substitute for typing.get_origin of Python 3.8+
-
-    Note that the input t must be one of the valid types permitted for
-    an input/attribute by ONNX Script.
-    """
-    if hasattr(typing, "get_origin"):
-        return typing.get_origin(t)
-    elif hasattr(t, "__origin__"):
-        return t.__origin__  # type: ignore[no-any-return]
-    else:
-        return None
-
-
-def _get_args(t: type) -> typing.Sequence[type]:
-    """Substitute for typing.get_args of Python 3.8+"""
-    if hasattr(typing, "get_args"):
-        return typing.get_args(t)
-    elif hasattr(t, "__args__"):
-        return t.__args__  # type: ignore[no-any-return]
-    else:
-        raise ValueError(f"Unsupported type annotation {t}")
-
-
-def _get_element_type(t: type) -> type:
-    """Returns the element type for a list or sequence type."""
-    return _get_args(t)[0]
-
-
 def pytype_to_attrtype(pytype: type) -> typing.Optional[onnx.AttributeProto.AttributeType]:
     if pytype in _pytype_to_attrtype_map:
         return _pytype_to_attrtype_map[pytype]
-    if _get_origin(pytype) in _list_constructors:
-        elt_type = _get_element_type(pytype)
+    if get_origin(pytype) in _list_constructors:
```

Comment:
Can `_list_constructors` be a frozenset?

Reply:
Not sure. It was a plain set originally, but there was warning messages about the elements being not hashable or something like that. I changed it to a list, since it shouldn't matter much for a 3-element collection. +1 for the naming convention, will do.

Reply:
I see. thanks for the context